web/themes/new_weather_theme/assets/js/charts/WeatherChart.js

```diff
@@ -30,5 +30,68 @@ const drawChart = (container, config) => {
   return new Chart(canvas, config);
 };
 
-export { drawChart };
-export default { drawChart };
+const setupScrollButtons = (container) => {
+  const isSynced = container.dataset.syncScrolling === "true";
+  const wrapper = container.closest('.wx-chart-wrapper');
+  const left = wrapper.closest('.wx-chart-wrapper').querySelector('.wx-scroll-button[data-direction="left"]');
+  const right = wrapper.querySelector('.wx-scroll-button[data-direction="right"]');
+  if(!left || !right){
+    return;
+  }
```

Comment:
Just a thought, but because this check is here, could we have the `drawChart` method also call `setupScrollButtons`? Then the only requirement to get the buttons to work is to include them in the twig template. No need to do anything with it here, but maybe a future refactor?

Reply:
I thought about that but didn't want to overload that one function (and was being cautious -- without looking -- in case it was a "real" 16x per second canvas draw function or whatever). We can definitely get this on the next loop around